openapi/event-reporting/api-spec.json

```diff
@@ -212,6 +212,12 @@
             "description": "A reference to the dated journey that is affected by this traffic event.",
             "example": "djj-4b60ddfd3a2aa810939912e08b76c58e"
           },
+          "vehicleInternalNumber": {
+            "type": "string",
+            "description": "The SIS internal number of the vehicle assigned to the journey and affected by this\nevent, if any.\n",
```

Comment:
I thought the operator internal number was unrelated to SIS? Don't this apply to non-SIS vehicles as well?

Reply:
Vi har av en eller annen historisk grunn kalt feltet `sisInternalNumber` ellers hos oss. Men ja, det er urelatert til SIS og gjelder alle kjøretøy. Har fjernet SIS-referansen